org.godotengine.Godot.appdata.xml

```diff
@@ -5,6 +5,10 @@
   <summary>Godot game engine editor</summary>
   <developer_name>The Godot Engine Community</developer_name>
   <description>
+    <p>
+      <strong>NOTE: This Flatpak is unofficial and not supported by Godot Engine developers.</strong>
```

Comment:
`<strong>` is not supported by the [AppStream spec](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-description), but `<em>` is for emphasis.

```suggestion
      <em>NOTE: This Flatpak is unofficial and not supported by Godot Engine developers.</em>
```